Makefile

```diff
@@ -67,4 +67,14 @@ distclean:
 	rm -rf tmp_install
 	cargo clean
 
+fmt:
+	@files=$$(git diff --cached --name-only --diff-filter=ACM | grep ".rs"); \
```

Comment:
We could reuse `../../pre-commit.sh.tpl` here (maybe with a different cmdline argument), couldn't we? Also, why is it called "template"? Nothing is substituted in this file.